spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/MaxoutSpec.scala

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
it should not be move to this package, keep the original one and updated the package info